website/sidebars.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
                 'topics/a-b-testing',
                 'topics/managing-constraints',
                 'topics/proxy-hosting',
+                'topics/data-collection',
```

Comment:
Suggest you move this up to before "managing-constraints" so that the list stays sorted alphabetically.

Reply:
Thank you, good call, done!